bin/tools/Cargo.toml

```
@@ -0,0 +1,6 @@
+[package]
+name = "photon-tools"
```

```suggestion
name = "photondb-tools"
```

Let's use the "photondb" prefix in this project.